data/categories.txt

```diff
@@ -9228,7 +9228,7 @@ el:^Καλύβα|αγροικία
 es:^Cabaña
 fi:^Autiotupa
 fr:^Cabane sauvage
-hu:^Kunyhó a vadonban
+hu:^Menedékház (személyzet nélkül)
```

Comment:
Note that this file contains synonyms for search: we will suggest the category if a user types in one of the strings separated by "|". Here someone needs to type in (almost exactly) ```Menedékház (személyzet nélkül)``` which I think is unlikely. Please try to find a shorted synonym: what would you type in the search bar if you wanted to find a wilderness hut?

Reply:
It can be just _menedékház_ ("személyzet nélkül" means "without personnel" which is not indispensable)